consensus/src/round_manager.rs

```diff
@@ -399,19 +399,20 @@ impl RoundManager {
     /// 1) In case a validator has previously voted in this round, it repeats the same vote and sign
     /// a timeout.
     /// 2) Otherwise vote for a NIL block and sign a timeout.
+    /// Note this function returns Err even if messages are broadcasted successfully because timeout
+    /// is considered as error. It only returns Ok(()) when the timeout is stale.
```

Comment:
Okay so timeouts are now Err, and everything downstream handles that?

Reply:
it's a high level function, the only downstream is that main loop